tachyon/zk/plonk/examples/fibonacci/fibonacci2_circuit.h

```diff
@@ -0,0 +1,134 @@
+#ifndef TACHYON_ZK_PLONK_EXAMPLES_FIBONACCI_FIBONACCI2_CIRCUIT_H_
+#define TACHYON_ZK_PLONK_EXAMPLES_FIBONACCI_FIBONACCI2_CIRCUIT_H_
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "tachyon/zk/plonk/constraint_system/circuit.h"
+
+namespace tachyon::zk::plonk {
+
+template <typename F>
+struct Fibonacci2Config {
+  using Field = F;
+
+  Fibonacci2Config(const AdviceColumnKey& advice, const Selector& selector,
```

Comment:
Please see the comment in selector.h

```suggestion
  Fibonacci2Config(const AdviceColumnKey& advice, Selector selector,
```